rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -251,7 +252,6 @@ SettingsDialog::SettingsDialog(wxWindow *parent, rpcs3::config_t* cfg)
 		chbox_gs_overlay->Enable(false);
 	}
 
-	cbox_gs_render->Append("Vulkan");
```

Comment:
Gotcha. I find it better to leave vulkan outside the MS check though if it's possible to use on linux, but I can throw it back in untill it runs OOFB